util/rlp/fuzz/fuzz_targets/append_raw.rs

```diff
@@ -0,0 +1,19 @@
+#![no_main]
+#![cfg(feature = "nightly")]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rlp;
+extern crate rand;
+
+use rlp::RlpStream;
+use rand::Rng;
+
+fuzz_target!(|data: &[u8]| {
+    let mut rls = RlpStream::new();
+
+    // Generate random item count to further fuzz append_raw's parsing
+    let mut rng = rand::thread_rng();
+    let rand_item_count = rng.gen_range::<usize>(1 as usize, 4096 as usize);
+
+    let _ = rls.append_raw(data, rand_item_count);
```

Comment:
`append_raw` doesn't do any checking of the data passed AFAIK. but it does check that `item_count` is within the number of remaining items in the list (which in this case, would be 1).

what do you expect to find by fuzz-testing this? `RlpStream` is typically not exposed to external input, whereas `UntrustedRlp` is.